src/main/java/net/sf/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -49,24 +49,24 @@
     private final FieldContentParser parser;
 
 
-    public LatexFieldFormatter() {
-        this(true);
+    public LatexFieldFormatter(JabRefPreferences prefs) {
+        this(true, prefs);
     }
 
-    private LatexFieldFormatter(boolean neverFailOnHashes) {
+    private LatexFieldFormatter(boolean neverFailOnHashes, JabRefPreferences prefs) {
```

Comment:
In case you need a set of configuration options from the preferences (as it is the case here), you could introduce a configuration object that takes all these configurations as parameters. By passing this object, you can avoid long parameter lists and get rid of the preferences dependency.